1.html

```diff
@@ -0,0 +1,208 @@
+					<li><a href="/rules">Rules</a></li>
+					<li>
+						<a
+							href="https://ourtech.community/coc"
+							target="_blank"
+							rel="noreferrer"
+							>Code of Conduct</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://t.me/OurTechComm"
+							target="_blank"
+							rel="noreferrer"
+							>Join us on Telegram</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/two-years"
+							target="_blank"
+							rel="noreferrer"
+							>Two Years of OTC</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/team"
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks #1</h1>
+		<h2>Supercharge your deployments with WASM</h2>
+		<p>
+			by
+			<a
+				href="https://twitter.com/rishit_dagli"
+				target="_blank"
+				rel="noreferrer"
+				>Rishit Dagli</a
+			>
+		</p>
+
+		<main id="main-content">
+			<h2 id="about">Abstract</h2>
+			<p>
+				Machine Learning inference (though what we talk about can be
+				applied to other computationally intensive tasks) is often a
+				computationally intensive task and could greatly benefit from
+				the speed of WebAssembly. What if I told you, that you could
+				improve the performance of your application deployed on Linux
+				containers.
+			</p>
+			<p>Enter Wasm and Kubernetes.</p>
+			<p>
+				This talk starts off by introducing the audience to WebAssembly
+				(Wasm) and how they could make use of the speed and security
+				among others of Wasm for their deployments. Another problem we
+				face is that the standard WebAssembly provides very limited
+				access to the native OS and hardware, such as multi-core CPUs,
+				GPU, or TPUS which is not ideal for the kind of systems we
+				target. The talk also shows how one could use the WebAssembly
+				System Interface (WASI) to get security, portability, and native
+				speed for Machine Learning models. The talk then slides into the
+				recent (still preview) support that Azure has launched for WASI
+				node pools to run such a system while also using Krustlet to run
+				Wasm on Kubernetes. To top it off this talk ends with a demo of
+				doing so on Azure.
+			</p>
+			<p>
+				The talk was followed by OTC CatchUp #111 —
+				<a
+					href="https://catchup.ourtech.community/summary/111"
+					target="_blank"
+					rel="noreferrer"
+					>Link to the Summary</a
+				>
+			</p>
+			<p>
+				Feel free to
+				<a
+					href="https://links.ourtech.community"
+					target="_blank"
+					rel="noreferrer"
+					>contact us</a
+				>.
+			</p>
+
+			<h2 id="when-where">When and Where</h2>
+			<ul>
+				<li><b>Date</b>: 24th December, 2022</li>
+				<li><b>Timing</b>: 10:30 PM IST</li>
+				<li><a href="/rules">Rules</a></li>
```

Comment:
`<li><b>Where</b>: Online mode</li>`

Reply:
Good catch, Pranav.

I think 👇 

```html
<li><b>Where</b>: Online before <a href="..." target="..." rel="...">OTC CatchUp</a></li>
```

is more informative.

Reply:
Yeah, this makes more sense.

Reply:
![image](https://user-images.githubusercontent.com/61280281/212723105-467388cf-c69b-4f3b-904f-603f623c98b6.png)

This?

Reply:
Looks good to me!